decoder/vaapidecoder_vp8.cpp

```diff
@@ -542,6 +544,13 @@ YamiStatus VaapiDecoderVP8::decode(VideoDecodeBuffer* buffer)
             status = ensureContext();
             if (status != YAMI_SUCCESS)
                 return status;
+            m_gotKeyFrame = true;
+        }
+        else {
+            if (!m_gotKeyFrame) {
+                //we can't decode frame without key
```

Comment:
1. I can add warning,
2. It's not needed by h264, h265. The parser will return error if no sps https://github.com/01org/libyami/blob/apache/codecparsers/h264Parser.cpp#L521
for vp9, I do not have enough time to fix it, maybe we can file a bug to track it.